public/locales/zh.json

```diff
@@ -0,0 +1,1003 @@
+{
+    "navigation": {
+        "home": "navigation.home = 首页",
```

Comment:
Please remove the translation key e.g. `navigation.home`, as it will show "navigation.home = 首页" in the final product instead of just "首页".

Same goes for the rest.